src/components/navigation-header/components/product-page-content/index.tsx

```diff
@@ -28,14 +30,21 @@ const ProductPageHeaderContent = () => {
 	return (
 		<>
 			<div className={s.productsDropdown}>
-				<NavigationHeaderDropdownMenu
-					ariaLabel="Main menu"
-					buttonClassName={s.productsDropdownButton}
-					dropdownClassName={s.productsDropdownPane}
-					itemGroups={allProductsItems}
-					leadingIcon={<IconHashicorp24 className={s.productsDropdownIcon} />}
-				/>
+				<NavigationMenu.Root className={s.mobileMenuNavList}>
```

Comment:
This is needed for the product panel being rendered in `<NavigationHeaderDropdownMenu />`.

Reply:
@RubenSandwich Are there `a11y` concerns with using `<NavigationMenu.Root />` where we're using it in this component?